.copier-answers.yml

```diff
@@ -19,6 +19,8 @@ org_name: Odoo Community Association (OCA)
 org_slug: OCA
 rebel_module_groups:
 - sale_packaging_default
+- sale_fixed_discount
+- sale_triple_discount,sale_order_general_discount_triple
```

Comment:
you can avoid this by adding ANOTHER glue module.... like I did in https://github.com/OCA/sale-workflow/pull/3907 but it hasn't really gathered much attention so it doesn't matter for now seperating is also ok